feeds/npm/npm.go

```diff
@@ -95,6 +91,8 @@ func fetchVersionInformation(packageName string) (string, error) {
 
 type Feed struct {
 	lossyFeedAlerter *feeds.LossyFeedAlerter
+	firehoseURL      string
```

Comment:
Might want to keep this called `baseURL`, seems more flexible and easier to understand if someone is unaware of usecase here of `firehose`

Reply:
I'm keen to rename these during these changes, perhaps not to firehose but the terminology of baseURL just seems incorrect as it's usually a URL containing the full path. I'll play around with the naming a little.

Reply:
baseURL is now actually the baseURL and the paths are added on where relevant. This makes hooking the tests up slightly more straight forward too since you can just provide the test server URL.